src/emp_agents/agents/base.py

```diff
@@ -183,15 +193,11 @@ async def _run_conversation(
                     tool_call.function.name,
                     tool_call.function.arguments,
                 )
-                message = Message(
-                    role=(
-                        Role.user
-                        if isinstance(model, AnthropicModelType)
-                        else Role.tool
-                    ),
+                # TODO - discuss, should tool results always map internally to tool messages?
```

Comment:
not sure - shouldn't we always use ToolMessage internaly even though Anthropic doesn't? Then when we send things back over the write we map our ToolMessages to UserMessages?